tasks/install-and-configure-packages.yml

```diff
@@ -49,6 +49,6 @@
       ha_cluster_hacluster_password
       | password_hash(
           'sha512',
-          ansible_hostname|replace('-','x')|truncate(16, True, '')
+          ansible_hostname|replace('-','x')|truncate(16, True, '', 0)
```

Comment:
This is problematic because of rhel7 controller support.  rhel7 uses jinja 2.7.  The truncate filter in jinja 2.7 does not have the leeway parameter: https://github.com/pallets/jinja/blob/2.7.3/jinja2/filters.py#L448

so I think we'll have to resort to using plain old python here
```yaml
# The user is created by installing pacemaker packages. We just need to set the
# password.
- name: Set hacluster password
  user:
    name: hacluster
    password: "{{
      ha_cluster_hacluster_password
      | password_hash(
          'sha512',  ansible_hostname.replace('-','x')[:16]
        )
    }}"
```

Reply:
Thanks for your advice, @richm. With your fix, the test passed. Updating the commit.